WordPress/Classes/Utility/Migration/ContentMigrationCoordinator.swift

```diff
@@ -12,21 +12,32 @@ class ContentMigrationCoordinator {
     private let dataMigrator: ContentDataMigrating
     private let userPersistentRepository: UserPersistentRepository
     private let eligibilityProvider: ContentMigrationEligibilityProvider
+    private let tracker: MigrationAnalyticsTracker
 
     init(coreDataStack: CoreDataStack = ContextManager.shared,
          dataMigrator: ContentDataMigrating = DataMigrator(),
          userPersistentRepository: UserPersistentRepository = UserDefaults.standard,
-         eligibilityProvider: ContentMigrationEligibilityProvider = AppConfiguration()) {
+         eligibilityProvider: ContentMigrationEligibilityProvider = AppConfiguration(),
+         tracker: MigrationAnalyticsTracker = .init()) {
         self.coreDataStack = coreDataStack
         self.dataMigrator = dataMigrator
         self.userPersistentRepository = userPersistentRepository
         self.eligibilityProvider = eligibilityProvider
+        self.tracker = tracker
     }
 
-    enum ContentMigrationCoordinatorError: Error {
+    enum ContentMigrationCoordinatorError: LocalizedError {
         case ineligible
         case exportFailure
         case localDraftsNotSynced
+
+        var errorDescription: String? {
+            switch self {
+            case .ineligible: return "Content export is ineligible"
+            case .exportFailure: return "Content export failed"
+            case .localDraftsNotSynced: return "Local drafts not synced"
+            }
```

Comment:
Not that it matters but I think general style if it exists is to put them on new lines 😀